### ISSUE TITLE
Better CSS and a11y on toolbar dropdown

### ISSUE THREAD

**Issue by Swiip** [ProtonMail/react-components#494]
Current toolbar dropdowns need improvements and does not match Angular v4 visually.

Here is a list of points to improve:
- React props and aria property to indicate a dropdown menu item currently active
- State of a dropdown menu item wich would be disabled but not the current one
- Review the margins of the dropdown container

Once this will be done, the temporary CSS fixes on `proton-mail` will be to remove.

**Comment by nico3333fr** [comment]
Food for though: basically, the current item should be show via `aria-current="true"` (which will be better vocalized) and can be styled/disabled from CSS (`pointer-events`, whatever).

Disabled state should be used only when the option is... disabled 🎉  (let's imagine there is item in the list, we could use disabled, or a feature that is not available depeding on the selected plan...)
